pages/common/sdcv.md

```diff
@@ -0,0 +1,33 @@
+# sdcv
+
+> StarDict command-line version, dictionary client.
+> Dictionaries are provided separately from the client.
+> More information: <https://manned.org/sdcv>.
+
+- Start sdcv interactively:
+
+`sdcv`
+
+- List installed dictionaries:
+
+`sdcv --list-dicts`
+
+- Display a definition from a specific dictionary:
+
+`sdcv --use-dict {{dictionary_name}} {{search_term}}`
+
+- Lookup a definition with a fuzzy search:
+
+`sdcv {{search_term}}`
+
+- Lookup a definition with an exact search:
```

Comment:
`Lookup` is a valid noun, but the verb should be `look up`. It's just the same as with `login` vs `log in`.